### ISSUE TITLE
Update docs with examples of error handling

### ISSUE THREAD

**Issue by elena-kolevska** [dapr/python-sdk#649]
## Describe the feature
We need to update the client docs with examples of error parsing, for the new standardised errors in dapr (https://github.com/dapr/dapr/pull/7257)

## Release Note
RELEASE NOTE: 
**DOCS UPDATE** Update docs with examples of error handling.

**Comment by berndverst** [comment]
There are no standardized errors yet - and these are unknown to me as maintainer. This issue is not actionable until standardized errors are released and finalized.

**Comment by elena-kolevska** [comment]
Yes, the issue is not actionable yet indeed, but I needed to create it so I could add it to the dapr/dapr PR, because they're related. The PR is not merged yet, but it's actively being worked on.
This is the work Cassie and I presented in the last community call btw, related to [this proposal](https://github.com/dapr/proposals/blob/main/0009-BCIRS-error-handling-codes.md).

**Comment by elena-kolevska** [comment]
Closed in https://github.com/dapr/python-sdk/pull/659
